src/main/rx/ghst_protocol.h

```diff
@@ -80,16 +86,35 @@ typedef union ghstFrame_u {
     ghstFrameDef_t frame;
 } ghstFrame_t;
 
-/* Pulses payload (channel data). Includes 4x high speed control channels, plus 4 channels from CH5-CH12 */
-typedef struct ghstPayloadPulses_s {
-    // 80 bits, or 10 bytes
+
+/* Pulses payload (channel data), for 4x 12-bit channels */
+typedef struct ghstPayloadServo4_s {
+    // 48 bits, or 6 bytes
     unsigned int ch1: 12;
     unsigned int ch2: 12;
     unsigned int ch3: 12;
     unsigned int ch4: 12;
+} __attribute__ ((__packed__)) ghstPayloadServo4_t;
+
+/* Pulses payload (channel data). Includes 4x high speed control channels, plus 4 channels from CH5-CH12 */
+typedef struct ghstPayloadPulses_s {
+    // 80 bits, or 10 bytes
+    ghstPayloadServo4_t ch1to4;
 
     unsigned int cha: 8;
     unsigned int chb: 8;
     unsigned int chc: 8;
     unsigned int chd: 8;
 } __attribute__ ((__packed__)) ghstPayloadPulses_t;
+
+/* Pulses payload (channel data), with RSSI/LQ, and other related data */
+typedef struct ghstPayloadPulsesRSSI_s {
+    // 80 bits, or 10 bytes
+   ghstPayloadServo4_t ch1to4;
+
+    unsigned int lq: 8;                 // 0-100
+    unsigned int rssi: 8;               // 0 - 128 sign inverted, dBm
+    unsigned int rfProtocol: 8;         
+    signed int txPwrdBm: 8;             // tx power in dBm, use lookup table to map to published mW values
+} __attribute__ ((__packed__)) ghstPayloadPulsesRSSI_t;
```

Comment:
```suggestion
} __attribute__ ((__packed__)) ghstPayloadPulsesRSssi_t;
```

Reply:
Probably meant ghstPayloadPulsesRssi_t 😄 
```suggestion
} __attribute__ ((__packed__)) ghstPayloadPulsesRssi_t;
```

Reply:
@haslinghuis: Not sure what you are talking about? :stuck_out_tongue_closed_eyes:

Reply:
Hmmm I swear I saw a capital `S` last night ;-) Oops there was some dust on my screen 😆